diff --git a/drw.c b/drw.c
index 4cdbcbe..39f58c6 100644
--- a/drw.c
+++ b/drw.c
@@ -140,11 +140,11 @@ xfont_create(Drw *drw, const char *fontname, FcPattern *fontpattern)
 	 * https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=916349
 	 * and lots more all over the internet.
 	 */
-	FcBool iscol;
-	if(FcPatternGetBool(xfont->pattern, FC_COLOR, 0, &iscol) == FcResultMatch && iscol) {
-		XftFontClose(drw->dpy, xfont);
-		return NULL;
-	}
+	//FcBool iscol;
+	//if(FcPatternGetBool(xfont->pattern, FC_COLOR, 0, &iscol) == FcResultMatch && iscol) {
+	//	XftFontClose(drw->dpy, xfont);
+	//	return NULL;
+	//}
 
 	font = ecalloc(1, sizeof(Fnt));
 	font->xfont = xfont;
